### PR TITLE
OZ-351: `net.mekomsolutions` → `com.ozonehis`.

### PR DESCRIPTION
**Pull request by corneliouzbett**
Issue: https://mekomsolutions.atlassian.net/jira/software/c/projects/OZ/issues/OZ-351

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,31 +3,53 @@
...
+            <url>https://nexus.mekomsolutions.net/repository/maven-public/</url>
+        </repository>
+    </repositories>
+
     <dependencies>
         <dependency>
-            <groupId>net.mekomsolutions.eip</groupId>
-            <artifactId>openmrs-eip-client-commons</artifactId>
+            <groupId>com.ozonehis.eip</groupId>
+            <artifactId>eip-client-commons</artifactId>
```
Comment (corneliouzbett) [inline pom.xml:34]: For group Id, should we go with just `com.ozonehis` or `com.ozonehis.eip`? we need to be consistent

Comment (corneliouzbett) [reply pom.xml:34]: I think the group Id should be `com.ozonehis`, then we add a package (eip) `com.ozonehis.eip.*` for all classes

File: pom.xml [hunk 2]
```diff
@@ -3,31 +3,53 @@
     xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>net.mekomsolutions</groupId>
-    <artifactId>eip-routes-senaite</artifactId>
+    <groupId>com.ozonehis</groupId>
+    <artifactId>eip-openmrs-senaite-routes</artifactId>
```
Comment (mks-d) [inline pom.xml:7]: → `eip-openmrs-senaite`

[This one](https://github.com/ozone-his/eip-keycloak-openmrs/blob/591df28266404359c153029ee686bfbce1840675/routes/pom.xml#L12) for Keycloak needs to be adapted as well to `eip-keycloak-openmrs`.

(The transient EIP Client there (pending the overall upgrade) will be named `eip-client-keycloak-openmrs`.)

**Comment by corneliouzbett** [comment]
@mks-d This is ready to be transferred. Cc @Ruhanga

**Review by Ruhanga — APPROVED** [review]

**Review by mks-d — APPROVED** [review]

**Pull request merged**